packages/template-retail-react-app/app/components/product-tile/index.test.js

```diff
@@ -108,3 +108,255 @@ test('Remove from wishlist cannot be muti-clicked', () => {
 })
+
+test('renders strike through price with variants product', () => {
+    const {getByText} = renderWithProviders(<ProductTile product={mockProductWithVariants} />)
+    expect(getByText(/black flat front wool suit/i)).toBeInTheDocument()
+    expect(getByText(/£191\.99/i)).toBeInTheDocument()
+    expect(getByText(/£320\.00/i)).toBeInTheDocument()
+})
+
+test('renders strike through price with set', () => {
+    const {getByText} = renderWithProviders(<ProductTile product={mockProductSet} />)
+    expect(getByText(/Winter Look/i)).toBeInTheDocument()
+    expect(getByText(/Starting at/i)).toBeInTheDocument()
+    expect(getByText(/£44\.16/i)).toBeInTheDocument()
+})
+
+test('renders strike through price with standard product', () => {
+    const {getByText} = renderWithProviders(<ProductTile product={mockStandardProduct} />)
+    expect(getByText(/Laptop Briefcase with wheels \(37L\)/i)).toBeInTheDocument()
+    expect(getByText(/£63\.99/i)).toBeInTheDocument()
+})
+
+const mockStandardProduct = {
+    currency: 'GBP',
+    hitType: 'product',
+    image: {
+        alt: 'Laptop Briefcase with wheels (37L), , large',
+        disBaseLink:
+            'https://edge.disstg.commercecloud.salesforce.com/dw/image/v2/ZZRF_001/on/demandware.static/-/Sites-apparel-m-catalog/default/dw7cb2d401/images/large/P0048_001.jpg',
+        link: 'https://zzrf-001.dx.commercecloud.salesforce.com/on/demandware.static/-/Sites-apparel-m-catalog/default/dw7cb2d401/images/large/P0048_001.jpg',
+        title: 'Laptop Briefcase with wheels (37L), '
+    },
+    orderable: true,
+    price: 63.99,
+    pricePerUnit: 63.99,
+    productId: 'P0048M',
+    productName: 'Laptop Briefcase with wheels (37L)',
+    productType: {
+        item: true
+    },
+    representedProduct: {
+        id: 'P0048M',
+        c_styleNumber: 'P0048',
+        c_tabDescription:
+            'Perfect for business travel, this briefcase is ultra practical with plenty of space for your laptop and all its extras, as well as storage for documents, paperwork and all your essential items. The wheeled system allows you to travel comfortably with your work and when you reach your destination, you can remove the laptop compartment and carry over your shoulder to meetings. It’s the business.',
+        c_tabDetails:
+            '1682 ballistic nylon and genuine leather inserts| Spacious main storage compartment for documents and binders|Removable, padded laptop sleeve with D-rings for carrying with shoulder strap|Change handle system and cantilever wheels|Zip pull in gunmetal with black rubber insert Leather “comfort” insert detailed handle|Internal storage pockets for CD-Rom and peripherals|Real leather inserts'
+    },
+    representedProducts: [
+        {
+            id: 'P0048M'
+        }
+    ],
+    tieredPrices: [
+        {
+            price: 63.99,
+            pricebook: 'gbp-m-list-prices',
+            quantity: 1
+        }
+    ]
+}
+const mockProductWithVariants = {
```

Comment:
This data may be a bit unique for this component since this data is part of product search. We don't use entire product search data for product tile test, so I keep it here